packages/sdk/src/resolveBinary.ts

```diff
@@ -31,67 +31,71 @@ export const engineEnvVarMap = {
 }
 export { BinaryType }
 export async function resolveBinary(name: BinaryType, proposedPath?: string): Promise<string> {
+  // if file exists at proposedPath (and does not start with `/snapshot/` (= pkg), use that one
   if (proposedPath && !proposedPath.startsWith('/snapshot/') && fs.existsSync(proposedPath)) {
     return proposedPath
   }
 
+  // If engine path was provided via env var, check and use that one
   const envVar = engineEnvVarMap[name]
-
   if (process.env[envVar]) {
     if (!fs.existsSync(process.env[envVar]!)) {
       throw new Error(`Env var ${envVar} is provided, but provided path ${process.env[envVar]} can't be resolved.`)
     }
     return process.env[envVar]!
   }
 
-  const dir = eval('__dirname')
-
+  // If still here, try different paths
   const binaryName = await getBinaryName(name)
 
-  let prismaPath = path.join(getEnginesPath(), binaryName)
+  const prismaPath = path.join(getEnginesPath(), binaryName)
   if (fs.existsSync(prismaPath)) {
     return maybeCopyToTmp(prismaPath)
   }
-  // for pkg
-  prismaPath = path.join(__dirname, '..', binaryName)
-  if (fs.existsSync(prismaPath)) {
-    return maybeCopyToTmp(prismaPath)
+  
+  // for pkg (related: https://github.com/vercel/pkg#snapshot-filesystem)
+  const prismaPath2 = path.join(__dirname, '..', binaryName)
+  if (fs.existsSync(prismaPath2)) {
+    return maybeCopyToTmp(prismaPath2)
   }
 
-  prismaPath = path.join(__dirname, '../..', binaryName)
-  if (fs.existsSync(prismaPath)) {
-    return maybeCopyToTmp(prismaPath)
+  // TODO for ??
```

Comment:
Looks like we check 2 levels up "for some reason"

Reply:
Yeah but what is the reason? :D I have no idea.